GOVERNANCE.md

```diff
@@ -0,0 +1,145 @@
+* Contribution policy
+* GitHub repository hosting
+* Conduct guidelines
+* Maintaining the list of additional Collaborators
+
+Initial membership invitations to the TC were given to individuals who
+had been active contributors to io.js, and who have significant
+experience with the management of the io.js project. Membership is
+expected to evolve over time according to the needs of the project.
+
+For the current list of TC members, see the project
+[README.md](./#current-project-team-members).
+
+## Collaborators
+
+The [iojs/io.js](https://github.com/iojs/io.js) GitHub repository is
+maintained by the TC and additional Collaborators who are added by the
+TC on an ongoing basis.
+
+Individuals making significant and valuable contributions are made
+Collaborators and given commit-access to the project. These
+individuals are identified by the TC and their addition as
+Collaborators is discussed during the weekly TC meeting.
+
+_Note:_ If you make a significant contribution and are not considered
+for commit-access log an issue or contact a TC member directly and it
+will be brought up in the next TC meeting.
+
+Modifications of the contents of the iojs/io.js repository are made on
+a collaborative basis. Anybody with a GitHub account may propose a
+modification via pull request and it will be considered by the project
+Collaborators. All pull requests must be reviewed and accepted by a
+Collaborator with sufficient expertise who is able to take full
+responsibility for the change. In the case of pull requests proposed
+by an existing Collaborator, an additional Collaborator is required
+for sign-off. Consensus should be sought if additional Collaborators
+participate and there is disagreement around a particular
+modification.
+
+Collaborators may opt to elevate significant or controversial
+modifications, or modifications that have not found consensus to the
+TC for discussion by assigning the ***tc-agenda*** tag to a pull
+request or issue. The TC should serve as the final arbiter where
+required.
+
+For the current list of Collaborators, see the project
+[README.md](./#current-project-team-members).
+
+A guide for Collaborators is maintained in
+[COLLABORATOR_GUIDE.md](./COLLABORATOR_GUIDE.md).
+
+## TC Membership
+
+TC seats are not time-limited.  There is no fixed size of the TC.
+However, the expected target is between 6 and 12, to ensure adequate
+coverage of important areas of expertise, balanced with the ability to
+make decisions efficiently.
+
+There is no specific set of requirements or qualifications for TC
+membership beyond these rules.
+
+The TC may add additional members to the TC by unanimous consensus.
+
+A TC member may be removed from the TC by voluntary resignation, or by
+unanimous consensus of all other TC members.
+
+Changes to TC membership should be posted in the agenda, and may be
+suggested as any other agenda item (see "TC Meetings" below).
+
+If an addition or removal is proposed during a meeting, and the full
```

Comment:
This is also problematic. It's going to be come very difficult to get the full TC in a meeting and if you can basically vote down this action by not attending we're open to pretty serious abuse because the person doesn't even have to show up and say why they disagree.